index.js

```diff
@@ -78,6 +78,10 @@ module.exports = class NoiseSecretStream extends Duplex {
     return streamId(handshakeHash, isInitiator, id)
   }
 
+  get keepAliveMs () {
```

Comment:
just rename the _property tp .keepAlive or make a method called getKeepAlive () if thats not possible

Reply:
Nice, that's much easier, it's renamed to `.keepAliveMs` now instead of `._keepAliveMs`

Reply:
without Ms

Reply:
Aha ok, done